tests/lib/packaging/arch.dockerfile

```
@@ -0,0 +1,17 @@
+FROM archlinux
+
+RUN pacman -Syu --noconfirm && \
+    pacman pacman -Suq --needed --noconfirm \
+        debugedit \
+        fakeroot \
+        git \
+        go \
+        go-tools \
+        xfsprogs \
+        python-docutils \
+        apparmor \
+        autoconf-archive \
+        squashfs-tools \
+        base-devel
```

we want to make sure that the packaging files are correct
```suggestion
        base-devel \
        git
```